velox/docs/functions/presto/conversion.rst

```diff
@@ -1203,6 +1281,34 @@ Examples:
   SELECT cast(ipprefix '1.2.3.4/24' as ipaddress) -- ipaddress '1.2.3.0'
   SELECT cast(ipprefix '2001:db8::ff00:42:8329/64' as ipaddress) -- ipaddress '2001:db8::'
 
+Cast to TDIGEST(DOUBLE)
+-----------------------
+
+From VARBINARY
+^^^^^^^^^^^^^^
+
+Returns a T-digest reconstructed from the varbinary string containing the serialized representation.
+This allows previously stored T-digests to be restored for use.
+
+::
+
+  SELECT cast(stored_tdigest_binary as tdigest(double)) FROM table_name;
```

Comment:
Is TDIGEST(DOUBLE) the only option allowed ? Since QDIGEST supports bunch of them ?